onnx/onnxifi_ext.h

```diff
@@ -0,0 +1,163 @@
+#ifndef ONNXIFI_EXT_H
+#define ONNXIFI_EXT_H 1
```

Comment:
#define ONNXIFI_EXT_H

should be enough

Reply:
Actually I am not quite sure about what is the meaning of this, just copying codes from Marat. Mind telling me why we should add this macro?

Reply:
this is to avoid redefinition in the same translation unit. So when you define those things `ONNXIFI_EXT_H`, header file will be included just once.